ol/cli/src/cache.rs

```diff
@@ -46,10 +47,20 @@ impl Vitals {
     /// reach the json cache
     pub fn read_json(node_home: &PathBuf) -> Vitals {
         let cache_path = get_cache_path(node_home);
-        let file = fs::File::open(cache_path).expect("file should open read only");
-        let deser: Vitals = serde_json::from_reader(file).expect("file should be proper JSON");
-
-        deser
+        let file = match fs::File::open(cache_path) {
+            Ok(f) => f,
+            Err(e) => {
+                println!("Could not open file: \nError {}", e);
+                exit(1)
```

Comment:
Ok this is a great start. In this case we actually don't want the program to always exit. So let's change the return of this function into a Result<> and refactor the callers of the function.

Reply:
So are we going to panic here and match in the callers?

Reply:
The two introduced unwraps are just for testing. There's something breaking the test